### PR TITLE
Localize sidebar component

### PR DESCRIPTION
**Pull request by alicoding**
@alanmoo R?

### CODE REVIEW

File: locales/en-US/messages.properties [hunk 1]
```diff
@@ -1 +1,15 @@
 MLN=The Mozilla Learning Network
+teaching_activities=Teaching ActivitiesL10N
```
Comment (alanmoo) [inline locales/en-US/messages.properties]: ^

**Comment by alanmoo** [comment]
Just fix that one string

File: locales/en-US/messages.properties (hunk 1 above)
Comment (alicoding) [inline locales/en-US/messages.properties]: I porgottttr

**Comment by alicoding** [comment]
@alanmoo done

**Pull request merged**